### DISCUSSION TITLE
How to train faces recognize?

### DISCUSSION THREAD

**Discussion by lehboi** [nextcloud/recognize#685] [Q&A] [answered]
Hi everyone,
I've just installed Recognize yesterday. Today I opened Photos and saw several clusters under People, also in Memories app.
I assume that each cluster containing photos of one person. I see that there are also photos not relating to that person in the cluster. How do I train Recognize to do better?
I did remove photos not relating to that person from the cluster. But after some cron jobs, the not-relating photos are added back to the cluster.
Thank you for your advices!
Boi

**Comment by marcelklehr** [comment, marked as answer]
> I did remove photos not relating to that person from the cluster. But after some cron jobs, the not-relating photos are added back to the cluster.

@lehboi That would be a bug. I'll fix it soon.
